Patch: 07_snooze.patch
Author: Tollef Fog Heen <tfheen@debian.org>
Snooze a bit to avoid eating too much CPU if you get an SMTP error (Closes: #175173)
Index: mailman-2.1.12/Mailman/Queue/OutgoingRunner.py
===================================================================
--- mailman-2.1.12.orig/Mailman/Queue/OutgoingRunner.py	2009-03-02 20:41:56.000000000 +0100
+++ mailman-2.1.12/Mailman/Queue/OutgoingRunner.py	2009-03-02 20:49:57.000000000 +0100
@@ -89,6 +89,7 @@
                 syslog('error', 'Cannot connect to SMTP server %s on port %s',
                        mm_cfg.SMTPHOST, port)
                 self.__logged = True
+            self._snooze(0)
             return True
         except Errors.SomeRecipientsFailed, e:
             # Handle local rejects of probe messages differently.
